db/seeds.rb

```diff
@@ -5,3 +5,20 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+
```

Comment:
Would it be advantageous for the first lines to destroy all existing term and lesson instances? I sometimes double-seed by accident, and it prevents errors like this: http://stackoverflow.com/questions/12103333/rails-reset-seeds-rb .